lib/galaxy/managers/licenses.py

```diff
@@ -1,10 +1,28 @@
 import json
 import logging
+from typing import List, Optional
```

Comment:
We typically do multiple imports from the same module like so:
```suggestion
from typing import (
    List,
    Optional
)
```

Reply:
Thanks Marius! Good to know :)